### PR TITLE
quazip: update to 1.4

### PR DESCRIPTION
**Pull request by ElHereward**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture **x86_64**

### CODE REVIEW

File: srcpkgs/quazip/template [hunk 1]
```diff
@@ -9,21 +9,16 @@ maintainer="Duncaen <duncaen@voidlinux.org>"
...
 quazip-devel_package() {
+	build_style=meta
 	depends="${sourcepkg}>=${version}_${revision}"
 	short_desc+=" - development files"
-	pkg_install() {
-		vmove usr/include
-		vmove usr/share
-		vmove "usr/lib/*.a"
-		vmove "usr/lib/*.so"
-	}
```
Comment (classabbyamp) [inline srcpkgs/quazip/template:26]: this is incorrect, just some files were moved around and the -devel package should not be empty.

```diff
@@ -20,8 +20,8 @@ quazip-devel_package() {
        short_desc+=" - development files"
        pkg_install() {
                vmove usr/include
-               vmove usr/share
-               vmove "usr/lib/*.a"
+               vmove usr/lib/cmake
+               vmove usr/lib/pkgconfig
                vmove "usr/lib/*.so"
        }
 }
```

**Comment by ElHereward** [comment]
@classabbyamp tx - the problem with vmove i could solve.
i changed the shlib template - but i dont understand what you mean by rebuild. Can i test it on my own?

**Comment by classabbyamp** [comment]
> i dont understand what you mean by rebuild

when the version of a shared library changes, the packages that use that shared library need to be rebuilt for the new version

> Can i test it on my own?

sure, but those packages also need to be included in this PR with the `revision` incremented

> why void packages quazip 0.8.1 but shlibs need 0.7.3?

`common/shlibs` records the _minimum_ version of the package that provides the shared library

**Comment by ElHereward** [comment]
> > i dont understand what you mean by rebuild
> 
> when the version of a shared library changes, the packages that use that shared library need to be rebuilt for the new version
> 
> > Can i test it on my own?
> 
> sure, but those packages also need to be included in this PR with the `revision` incremented
> 
> > why void packages quazip 0.8.1 but shlibs need 0.7.3?
> 
> `common/shlibs` records the _minimum_ version of the package that provides the shared library

okay tx - then i need to read the docs twice :D

**Pull request closed without merging**